README.md

```diff
@@ -333,6 +334,10 @@ Note that: the escape character is `＼` (U+FF3C) not `\`.
 
 For normal ASCII characters, you can just put them into the input field.
 
+### Conway's Life of Game
+
+See [tools/qftasm/README.md](tools/qftasm/README.md) for its details.
+
 ##### Execute it from command line
```

Comment:
This subsection seems to be from the Scratch section, the one before the Game of Life section.

Reply:
Oops, thanks!!